File-Formats/PDF/Convert-HTML-To-PDF/docker.md

```diff
@@ -7,91 +7,89 @@ documentation: UG
 keywords: Assemblies
 ---
 
+# Convert HTML to PDF file in Linux docker container
+
 The Syncfusion HTML to PDF converter is a .NET library that converts HTML or web pages to PDF.  This library converts an HTML to PDF in a Linux [docker](https://www.docker.com/why-docker/) container.
```

Comment:
The Syncfusion HTML to PDF converter is a .NET library that converts HTML or web pages to PDF document in a Linux [Docker](https://www.docker.com/why-docker/) container.

Reply:
content changed